release-1.5/release-notes-draft.md

```diff
@@ -0,0 +1,104 @@
+# v1.5.0 (draft)
+
+[Documentation](http://kubernetes.github.io) & [Examples](http://releases.k8s.io/release-1.5/examples)
+
+## Major Themes
+
+- Improved Federation Support
+  - Daemonsets
+  - Deployments
+  - ConfigMaps
+- Simplified Cluster Deployment
+  - `kubeadm`
+  - `kubefed`
```

Comment:
maybe: `kubefed`: new binary

or something that makes it clear that kubefed is a new binary and kubeadm has just been improved